beets/autotag/mb.py

```diff
@@ -571,7 +590,7 @@ def match_track(artist, title):
         yield track_info(recording)
 
 
-def _parse_id(s):
+def _parse_id(s: str) -> Optional[Union[str, bytes]]:
```

Comment:
I don't think this can return `bytes`, can it?

Reply:
No it cannot, not sure why I did that.